drivers/Kconfig

```diff
@@ -9,6 +9,8 @@
 
 menu "Device Drivers"
 
+source "drivers/1-wire/Kconfig"
```

Comment:
After applying the patch to current tree and resolving patch errors, CMake errors out on build because it can't find the above directory. Looks like this was renamed as `drivers/w1/Kconfig` down below. Removing this line resolves that.

Reply:
Fixed